main.py

```diff
@@ -2,6 +2,33 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George Klooney", "Kianu Reaves", "Scarlett Keegan"),
```

Comment:
The actors' names are incorrectly grouped in tuples. Each actor should be represented as a separate tuple with first and last names, e.g., ('George', 'Klooney').